### ISSUE TITLE
A Basic Example for formRender

### ISSUE THREAD

**Issue by DarwinJS** [kevinchappell/formBuilder#1511]
### Description: 

The basic example for formBuilder makes a lot of sense - a complete page which implements the builder, including the js includes and a special named div id: https://formbuilder.online/docs/getting-started/#basic-example

However, the basic example for formRender seems to lack the completeness of showing a ready to go webpage with the js includes for rendering and any special div ids.

Basically the online builder you provide is sufficient for my purposes - but needing to understand how to setup formRender for simple static forms.

**Comment by lucasnetau** [comment]
Pull Request are welcome for documentation updates

There are demo examples of using formRender 
https://formbuilder.online/docs/formRender/demos/basic/
https://formbuilder.online/docs/formRender/options/formData/

for formRender all you need is a container (div or form element) and to call the formRender function with the form configuration.
